CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+<a name="v2.0.2"></a>
+## [v2.0.2](https://github.com/solid/community-server/compare/v2.0.1...v2.0.2) - 2022-01-20
+
+### Added
+* [feat: Offical Docker image build](https://github.com/solid/community-server/commit/48bb9c2cca43e683d66f6a3eff921b59787bfb9d)
+
```

Comment:
Never wrote changelogs here before. Is this supposed to be a new patch version. Or did I do this wrong?

Reply:
@joachimvh Can help you out.

Reply:
It should actually be a RELEASE_NOTES entry, the CHANGELOG entries are (semi-)automatically generated on release.

It is a bit weird to do a new release for this though, since this doesn't actually change anything about the server itself. Perhaps it would make more sense to include this in the 3.0.0 release notes as the plan is to release that version soon-ish?

Reply:
perfectly fine for me!

Reply:
We could always do a patch though, there are some things on `main` still.

Reply:
People might be less inclined to check the notes of a patch release. Either is fine for me though, don't have a strong opinion.

Reply:
(might still want to merge it on `main` to already have the CI running)

Reply:
> (might still want to merge it on `main` to already have the CI running)

That's also what I was thinking, merge this in main and create a separate commit in the 3.0.0 branch indicating the new feature.